Assets/Scripts/Internal/DaggerfallGC.cs

```diff
@@ -0,0 +1,40 @@
+// Project:         Daggerfall Tools For Unity
+// Copyright:       Copyright (C) 2009-2020 Daggerfall Workshop
+// Web Site:        http://www.dfworkshop.net
+// License:         MIT License (http://www.opensource.org/licenses/mit-license.php)
+// Source Code:     https://github.com/Interkarma/daggerfall-unity
+// Original Author: Pango (petchema@concept-micro.com)
+// Contributors:    
+// 
+// Notes:
+//
+
+using System;
+using UnityEngine;
+
+namespace DaggerfallWorkshop
+{
+    public class DaggerfallGC
+    {
+        // Min time between two unused assets collections
+        private const float uuaThrottleDelay = 180f;
+
+        private static float uuaTimer = Time.realtimeSinceStartup;
+
+        private DaggerfallGC()
+        {
+        }
+
```

Comment:
Is there any reason you used a private constructor that never get called instead of a static class or you just preferred this way?

Reply:
Oops, just Java habits!